### PR TITLE
Closes #21 - Enable / Disable CDN with the specified host and zone.

### LINKED ISSUE #21: Ability to turn on an off CDN
It would be great if you can set the CDN host as well as turning CDN caching on and off

### PR DESCRIPTION
**Pull request by crystinutzaa**
Closes https://github.com/wp-media/wp-rocket-cli/issues/21

Enables / Disables CDN option with host and zone. It merges if the cname exists so it will change the zone.

My question is if we shall clean Rocket cache and minified files after this operation directly from CLI cdn command.
@wp-media/php

### CODE REVIEW

File: command.php [hunk 1]
```diff
@@ -300,6 +300,71 @@ public function regenerate( $args = array(), $assoc_args = array() ) {
...
+	 *
+	 *     wp rocket cdn --enable=false
+	 *     wp rocket cdn --enable=true --host=http://cdn.example.com
+	 *     wp rocket cdn --enable=true --host=http://cdn.example.com --zone=all
+	 *
+	 * @subcommand cdn
+	 */
+	public function cdn( $args = array(), $assoc_args = array() ) {
+		if ( empty( $assoc_args['enable'] ) ) {
+			WP_CLI::error( 'You did\'t specify the "enable" argument.' );
```
Comment (remyperona) [inline command.php]:
```suggestion
			WP_CLI::error( 'The "enable" argument must be specified.' );
```

**Review by remyperona — CHANGES_REQUESTED** [review]

**Comment by remyperona** [comment]
I think it would make sense to do the clean, as it can be easy to forget when changing just this value, and it can create errors if not done.

**Review by remyperona — APPROVED** [review]

**Comment by crystinutzaa** [comment]
@wp-media/qa  this one is ready for QA

**Comment by piotrbak** [comment]
@Tabrisrp @crystinutzaa It's expected to remove the zones in the `epic` here:
https://github.com/wp-media/wp-rocket/issues/3417

Should we take it into consideration with this update and allow only for the `all` here?

**Comment by remyperona** [comment]
The zone value itself won't exist anymore, but we don't know when this will be implemented, so we can go ahead with this for now. We will update it later at the same time we work on the epic.

**Comment by crystinutzaa** [comment]
@wp-media/qa please revisit this and approve it if everything seems ok. Thanks!

**Comment by piotrbak** [comment]
@Tabrisrp @crystinutzaa Check my comments:
1. We should have a possibility to remove the specific host/zone settings
2. We should have a possibility to list the current CDN settings (that will make automation easier)
3. It doesn't include multisite compatibility, we can only enable CDN for the main site
4. Also, when adding another CDN for all assets, shouldn't we delete the first one? Right now it's just adding another below the first. Without the possibility of removal of the host/zone settings, we can have a big mess there
5. When zone is not correct, we're enabling it for all assets. We should either not enable it at all or output a message that the CDN was enabled for all assets:
image

The happy scenarios work correctly.

**Comment by remyperona** [comment]
Thank you for the feedbacks Piotr.

I'd say, let's go with this MVP version of the command, as a start, and list your additional points as new issues to be worked on later.

I believe it's better to iterate small, especially in this case for a tool that is a side helper.

**Pull request merged**